docs/modules/graphics.md

```diff
@@ -22,6 +22,8 @@ The `Canvas` class is the core api for graphical display.
 This is the height of the canvas/viewport, in pixels.
 #### `static width: Number`
 This is the width of the canvas/viewport, in pixels.
+#### `static font: String`
```

Comment:
Oops, There's actually already documentation for this field, it's just in the wrong place. Let's move it up to here (and preferably keep the fields in alphabetical order).